packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -247,6 +247,17 @@ export const fillCustomDestinationAddress = () => {
     .typeRecursively(Cypress.env('CUSTOM_DESTINATION_ADDRESS'))
 }
 
+export function typeAmount(
+  amount: string | number
+): Cypress.Chainable<JQuery<HTMLElement>> {
+  const amountStringified = String(amount)
+
+  return cy
+    .findByPlaceholderText(/enter amount/i)
+    .typeRecursively(amountStringified)
+    .should('have.value', amountStringified)
+}
```

Comment:
Looking at failures on input typing, it's usually because we type `0.000X`, and if the `.` doesn't register properly, and all other 0 are ignored. Can we maybe type `.`, wait for the `.` to be in the input, then type rest of the amount?

Reply:
I don't think that would change anything, because if a different character is ignored that would fail as well